src/model/openthaigpt_pretraining_model/GPTJ_TH_tokenizer/merge.py

```diff
@@ -1,27 +1,13 @@
 from transformers import AutoTokenizer, GPT2TokenizerFast
-from huggingface_hub import hf_hub_download
-from .constants import FILE_DIR, GPT2_MERGE_DIR, GPTJ_MERGE_DIR
```

Comment:
Please remove unused constants in `constants.py` too.

Reply:
done